codec-http2/src/main/java/io/netty/handler/codec/http2/Http2ConnectionHandler.java

```diff
@@ -617,7 +617,7 @@ public ChannelFuture resetStream(final ChannelHandlerContext ctx, int streamId,
         }
 
         final ChannelFuture future;
-        if (stream.state() == IDLE || connection().local().created(stream)) {
```

Comment:
sgtm ... I knew you were working on not-queuing initial headers, but I wasn't sure it had gone in yet. Thanks!